README.md

```diff
@@ -208,11 +208,11 @@ To see the complete implementation of the above, refer [bookstore_service.bal](h
 
 - Navigate to `messaging-with-jms-queues/guide` and run the following commands in separate terminals to start both the JMS producer `bookstoreService` and  JMS consumer `orderDeliverySystem`.
 ```bash
-   $ ballerina run bookstore_service
+   $ ballerina run bookstore_service.bal
```

Comment:
Do we need these changes? We are running the package right?

Reply:
Since we've asked to do a ballerina `init` first and then navigate to `messaging-with-jms-queues/guide` and do ballerina `run`, IMO these changes are not needed?